packages/webpack/src/config/base.js

```diff
@@ -20,16 +20,9 @@ import WarnFixPlugin from '../plugins/warnfix'
 import { reservedVueTags } from '../utils/reserved-tags'
 
 export default class WebpackBaseConfig {
-  constructor(builder, options) {
-    this.name = options.name
-    this.isServer = options.isServer
-    this.isModern = options.isModern
+  constructor(builder) {
     this.builder = builder
-    this.nuxt = builder.context.nuxt
-    this.isStatic = builder.context.isStatic
-    this.options = builder.context.options
-    this.loaders = this.options.build.loaders
-    this.buildMode = this.options.dev ? 'development' : 'production'
+    this.context = builder.context
```

Comment:
`this.context` name is still ambiguous. Maybe we can call it `this.builderContext`?

Reply:
OK